### GOAL DEFINITION

# Path Planning
geometry_msgs/PoseArray path

# Average velocity of the route ([meter]/[second])
float32 average_velocity

# Sampling time ([second])
float32 sampling_time
---
### RESULT DEFINITION

# Reference State Matrix in array form [x y dx dy ddx ddy]
#
# |   x    |
# |   y    |
# |   dx   |
# |   dy   |
# |   ddx  |
# |   ddy  |

# Data organized in row-major order, starting with (0,0)
std_msgs/Float32MultiArray data

int32 row_size
int32 column_size